docs/monitoring/faq/webhooks.mdx

```diff
@@ -77,3 +77,116 @@ When delivering events to your webhook endpoint we follow these guidelines
 - If your endpoint returns a 5xx status code in less than 5 seconds we retry up to 2 times with exponential backoff.
```

Comment:
There are some cases where a Modal web endpoint might not spin up and ACK in less than 5 seconds. To avoid that, you can put `keep_warm` on the `stub.function` so you don't hit cold boots, but then you lose autoscaling. You can also reduce the chance of by putting less work in the path of the request -- e.g. defining a separate `stub.function` to handle a `run` and using `Function.spawn` to launch it as a background task, which means you can get to the ACK faster.

I'll include that in my example